src/arch/aarch64/CodeGen.zig

```diff
@@ -4695,11 +4684,22 @@ fn genTypedValue(self: *Self, typed_value: TypedValue) InnerError!MCValue {
 
             return self.lowerUnnamedConst(typed_value);
         },
-        .Struct => {
-            return self.lowerUnnamedConst(typed_value);
-        },
-        else => return self.fail("TODO implement const of type '{}'", .{typed_value.ty.fmtDebug()}),
+
+        .ComptimeInt => unreachable, // semantic analysis prevents this
+        .ComptimeFloat => unreachable, // semantic analysis prevents this
+        .Type => unreachable,
+        .EnumLiteral => unreachable,
+        .Void => unreachable,
```

Comment:
FYI https://github.com/ziglang/zig/blob/b08d32ceb5aac5b1ba73c84449c6afee630710bb/src/arch/x86_64/CodeGen.zig#L6766 This will resurface when working with error unions and implicit `return 0;` for `anyerror!void` return types AFAIK.

Reply:
Ok, should I change it now or later?

Reply:
Up to you, just letting you know what it currently looks like :-)